File: test/Condition.test.ts
import {expect} from 'chai';
import {Condition} from "../src/condition/Condition";

describe('Condition test', () => {
  
  let condition: Condition;
  
  before(() => {
    condition = new Condition();
  });
  
  it('It should return a > b', () => {
    expect(condition.compare(10, 1)).to.equal(">");
  });
  
  it('It should return a < b', () => {
    expect(condition.compare(1, 10)).to.equal("<");
  });
  
  it('It should return a = b', () => {
    expect(condition.compare(10, 10)).to.equal("=");
  });
});

File: test/src/ConditionTest.kt
import condition.Condition
import org.testng.annotations.Test
import org.testng.AssertJUnit.assertEquals

class ConditionTest{
  var condition: Condition = Condition()

  @Test
  fun `It should return a greater than b`(){
    assertEquals(condition.compare(10, 1), ">")
  }

  @Test
  fun `It should return a less than b`(){
    assertEquals(condition.compare(1, 10), "<")
  }

  @Test
  fun `It should return a equals to b`(){
    assertEquals(condition.compare(10, 10), "=")
  }
}